Neos.Neos/Classes/Fusion/Helper/BackendHelper.php

```diff
@@ -35,8 +35,6 @@ class BackendHelper implements ProtectedContextAwareInterface
      */
     public function interfaceLanguage(): string
     {
-        $currentUser = $this->userService->getBackendUser();
```

Comment:
the fluid viewhelper `neos:backend.translate` could also be used in the frontend - thats why we are not so strict over here.